audit/pom.xml

```diff
@@ -22,9 +22,9 @@
             <version>0.0.1-SNAPSHOT</version>
         </dependency>
         <dependency>
-            <groupId>org.quartz-scheduler</groupId>
-            <artifactId>quartz</artifactId>
-            <version>${quartz.version}</version>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-quartz</artifactId>
+            <version>2.2.1.RELEASE</version>
```

Comment:
We should probably drop this to make sure Spring Boot version is inherited from the top-level POM. This line says `2.2.1` while the top-level POM still refers to `2.2.0`.